ansible/roles/esdc-mgmt/tasks/esdc.yml

```diff
@@ -37,15 +37,17 @@
     - libjpeg-devel
     - patch
 
-- name: Download GeoIP database
-  get_url: url="http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz"
-           dest="/tmp/GeoIP.dat.gz"
-
-- name: Extract GeoIP database
-  shell: gzip -cd /tmp/GeoIP.dat.gz > /usr/share/GeoIP/GeoIP.dat
-
-- name: Remove downloaded GeoIP database
-  shell: rm -f /tmp/GeoIP.dat.gz
+# GeoIP DB is no longer downloadable and the DB was updated in rpm itself
```

Comment:
Why to keep it here as comments?
(this is a version control system)

Reply:
Because it will need to be addressed some time in the future. It's a reminder.